src/util/validateConfig.js

```diff
@@ -1,14 +1,21 @@
 import log from './log'
 
 export function validateConfig(config) {
-  if (config.content.files.length === 0) {
+  if (config.content.files !== 'auto' && config.content.files.length === 0) {
```

Comment:
It seems this change is unnecessary, since `files.length === 0` invalidates `!== 'auto'`